### ISSUE TITLE
Could you provide the arguments or checkpoints?

### ISSUE THREAD

**Issue by raretomato** [JohnsonSign/MaST-Pre#4]
I trained the model using the same default arguments as provided in 0-pretraining-dp.py. However, the learning process did not converge, and even after changing the masking ratio to 65% and 85%, there was no significant difference in the average point loss. Additionally, the results differ from the Accuracy provided in Table 5. Could you kindly provide the arguments you used or the pre-trained checkpoint?

**Comment by JohnsonSign** [comment]
Apologies for the delay. We've provided two new logs: “https://drive.google.com/drive/folders/1p0U4sF11lPrJvI8ZSwrmdBgJAx5P0rF4?hl=zh-cn.” 
In our subsequent research, we observed that this method exhibits fluctuations, possibly due to noise and the process of restoring the original point coordinates. Additionally, there is a notable gap between restoring these coordinates and learning high-level semantics. As a result, the 4D self-supervised task requires further optimization, and additional in-depth research is necessary.
